webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java

```diff
@@ -9,6 +9,7 @@
 import java.util.Map;
 import java.util.Properties;
 import java.util.Set;
+import java.util.Vector;
```

Comment:
It is a synchronized list and the code is copied from transferCopyEntities method at SiteAction, added in SAK-16568. I can replace it by ArrayList for a not synchronized solution but note that SiteAction remains using synchronized lists.